superset-frontend/src/explore/components/PropertiesModal.tsx

```diff
@@ -44,9 +67,9 @@ function PropertiesModalWrapper({ show, onHide, animation, slice, onSave }) {
   );
 }
 
-function PropertiesModal({ slice, onHide, onSave }) {
+function PropertiesModal({ slice, onHide, onSave }: InternalProps) {
   const [submitting, setSubmitting] = useState(false);
-  const errorDialog = useRef();
+  const errorDialog = useRef<any>(null);
```

Comment:
I take it this could be anything, and we can't safely use `<HTMLWhateverElement>` here?

Reply:
Yeah I don't think it has a defined type.